### PR TITLE
missing full pdfs emnlp

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/xml/2021.findings.xml [hunk 1]
```diff
@@ -6098,6 +6098,7 @@
       <address>Punta Cana, Dominican Republic</address>
       <month>November</month>
       <year>2021</year>
+      <url hash="e0ab15f8">2021.findings-emnlp</url>
```
Comment (mjpost) [inline data/xml/2021.findings.xml:6101]: This PDF is also not on the server.

File: data/xml/2020.findings.xml [hunk 2]
```diff
@@ -10,6 +10,7 @@
       <address>Online</address>
       <month>November</month>
       <year>2020</year>
+      <url hash="631fdb62">2020.findings-emnlp</url>
```
Comment (mjpost) [inline data/xml/2020.findings.xml:13]: @xinru1414 This PDF is not on the server.

**Comment by xinru1414** [comment]
@mjpost i double checked they are there and the hash match.

**Comment by mjpost** [comment]
My bad. I was in the `emnlp` directory and searched for `emnlp-findings` (so had it backwards).

Thanks!

**Review by mjpost — APPROVED** [review]

**Pull request merged**